docs/source/tracking/server.rst

```diff
@@ -0,0 +1,247 @@
+======================
+MLflow Tracking Server
+======================
+
+MLflow tracking server is a stand-alone HTTP server that serves multiple REST API endpoints for tracking runs/experiments.
+While MLflow Tracking can be used in local environment, hosting a tracking server is powerful in the team development
+workflow:
+
+* **Collaboration**: Multiple users can log runs to the same endpoint, and query runs and models logged by other users.
+* **Sharing Results**: The tracking server also serves :ref:`tracking_ui` endpoint, where team members can easily explore each other's results.
+* **Centralized Access**: The tracking server can be run as a proxy for the remote access for metadata and artifacts, making it easier to secure and audit access to data.
+
+Start the Tracking Server
+=========================
+
+Starting the tracking server is as simple as running the following command:
+
+.. code-block:: bash
+
+    mlflow server --host 127.0.0.1 --port 8080
+
+Once the server starts runing, you should see the following output:
+
+.. code-block::
+
+  [2023-11-01 10:28:12 +0900] [28550] [INFO] Starting gunicorn 20.1.0
+  [2023-11-01 10:28:12 +0900] [28550] [INFO] Listening at: http://127.0.0.1:8080 (28550)
+  [2023-11-01 10:28:12 +0900] [28550] [INFO] Using worker: sync
+  [2023-11-01 10:28:12 +0900] [28552] [INFO] Booting worker with pid: 28552
+  [2023-11-01 10:28:12 +0900] [28553] [INFO] Booting worker with pid: 28553
+  [2023-11-01 10:28:12 +0900] [28555] [INFO] Booting worker with pid: 28555
+  [2023-11-01 10:28:12 +0900] [28558] [INFO] Booting worker with pid: 28558
+  ...
+
+There are many options to configure the server, refer to :ref:`Configure Server <configure-server>` for more details.
+
+.. important:: 
+  The server listens on http://localhost:5000 by default and only accepts
+  connections from the local machine. To let the server accept connections
+  from other machines, you will need to pass ``--host 0.0.0.0`` to listen on
```

Comment:
Can we mention that this is a required conf for deployment inside k8s / docker containers (generally)?

Reply:
Sure, adding real use case should help users (+ searchability)